js/strings/da_dk.js

```diff
@@ -34,7 +34,7 @@ s.writeCommentHelpText = "Hvis dit perspektiv ikke er repræsenteret, så <stron
 s.helpWriteListIntro = "Hvad er en god kommentar?";
 s.helpWriteListStandalone = "Enkeltstående idé";
 s.helpWriteListRaisNew = "Rejser nye perspektiver, erfaringer eller problemstillinger";
-s.helpWriteListShort = "Klar og præcis (begrænset til 140 tegn)";
+s.helpWriteListShort = "Klar og præcis (begrænset til 280 tegn)";
```

Comment:
@urakagi This seems like another config file setting (in order to facilitate easier code sharing between forks).

Reply:
Sure.